stable/artifactory-ha/README.md

```diff
@@ -597,25 +597,24 @@ jconsole artifactory-ha-<release-name>-primary:<primary-jmx-port>
 jconsole <release-name>:<node-jmx-port>
 ```
 
-### Access creds. bootstraping
-**IMPORTANT:** Bootsrapping access creds. will allow access for the user access-admin from certain IP's.
+### Bootstrapping Artifactory admin password
+You can bootstrap the `admin` user password as described in the [bootstrap Artifactory admin credentials](https://www.jfrog.com/confluence/display/JFROG/Users+and+Groups#UsersandGroups-RecreatingtheDefaultAdminUserrecreate) guide.
 
-* User guide to [bootstrap Artifactory Access credentials](https://www.jfrog.com/confluence/display/ACC/Configuring+Access)
-
-1. Create `access-creds-values.yaml` and provide the IP (By default 127.0.0.1) and password:
+1. Create `admin-creds-values.yaml` and provide the IP (By default 127.0.0.1) and password:
 ```yaml
 artifactory:
   accessAdmin:
```

Comment:
Does it make sense to change the name of the key from `accessAdmin` to `admin` ?

Reply:
Yes, it does. I kept it for compatibility...

Reply:
Well, but this is a different thing... Might be worth breaking compatibility to make it more understandable. Also, I doubt this is a popular feature, so it probably won't be felt by most of the users